fc/jira/issue.py

```diff
@@ -76,6 +76,9 @@ def _extra_json_for_create(self, existing_json: dict):
     def type_str(self) -> str:
         raise NotImplementedError
 
+    def update_vfr(self, duration: int, cost_of_delay: int) -> float:
```

Comment:
Actually, shouldn't we just use the existing `score` method?  So, `BacklogStory` has a `set_duration` method and a `set_cost_of_delay` method.  When `fc backlog vfr ...` (should it be `fc backlog score ...` to keep the same nomenclature?) is called, it would call `Issue.get_issue`, return the correct `BacklogStory`, you call `set_duration` and `set_cost_of_delay`, and then subsequently call the `score` method and it'll do the correct thing.  Then we don't even need this `update_vfr` method.

Reply:
hmm... let me think on that a bit

Reply:
Implemented as described